src/ol/format/gml/gmlbaseformat.js

```diff
@@ -68,18 +68,36 @@ ol.format.GMLBase = function(opt_options) {
    */
   this.schemaLocation = '';
 
+  /**
+   * @type {Object.<string, Object.<string, Object>>}
+   */
+  this.FEATURE_COLLECTION_PARSERS = Object({
+    'http://www.opengis.net/gml': {
```

Comment:
And the use of the `Object` constructor is not necessary anymore. It was there to fool `gjslint`.